src/ui/profile_preferences.ui

```diff
@@ -156,10 +156,10 @@
           <item>
            <widget class="QCheckBox" name="mRawStreamDump">
             <property name="toolTip">
-             <string>Switch log file format from HTML to raw stream dump in logfiles as sent by server. The logfile can then be can be loaded by Mudlet to test your trigger systems offline.</string>
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked will cause the date-stamp named log file to be HTML (file extention '.html') which can convey color, font and other formatting information rather than a plain text (file extension '.txt') format.  It is not recommended to change this setting whilst logging is active!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
"It is not recommended to change this setting whilst logging is active!" why? This shouldn't do anything while it is active and only take effect for the next log.

People are bound to do something that is possible to do, so someone will stuff it up and be upset their logging got stuffed up.